scripts/check_ecr_findings

```diff
@@ -9,7 +9,7 @@ function_qualifier="$3"
 payload="$(jq --null-input ".repository= \"$repository\" | .imageTag = \"$image_tag\"")"
 aws lambda invoke --function-name ecr-scan-findings --cli-binary-format raw-in-base64-out --payload "$payload" --qualifier "$function_qualifier" scan_response.json
 jq < scan_response.json
-totalFindings="$(jq "fromjson | .totalFindings" scan_response.json)"
```

Comment:
What happened here?

Reply:
I changed the lambda's return type from a string to an object. It makes the code cleaner in the lambda, as well as the output returned to the caller.

previously:
func evaluateImage() (string, error)

now:
func evaluateImage() (ecrscan.Report, error)

previous response:

./scripts/check_ecr_findings "easi-backend" "$CIRCLE_SHA1" "1"
{
    "StatusCode": 200,
    "ExecutedVersion": "1"
}
"{\"totalFindings\":0}"


new response:

./scripts/check_ecr_findings "easi-backend" "$CIRCLE_SHA1" "3"
{
    "StatusCode": 200,
    "ExecutedVersion": "3"
}
{
  "totalFindings": 0
}